File: articles/knowledge-architecture-in-sustainability-science/cpp/sustainability_metrics.cpp
#include <iostream>
#include <iomanip>

int main() {
    double objects = 10.0;
    double relationships = 11.0;

    std::cout << std::fixed << std::setprecision(3);
    std::cout << "Metadata coverage: " << 9.0 / objects << "\n";
    std::cout << "Justice context coverage: " << 6.0 / objects << "\n";
    std::cout << "Uncertainty context coverage: " << 8.0 / objects << "\n";
    std::cout << "Relationship traceability: " << 10.0 / relationships << "\n";
    std::cout << "Underspecified relationship risk: " << 1.0 / relationships << "\n";

    return 0;
}

File: articles/knowledge-architecture-in-sustainability-science/fortran/sustainability_metrics.f90
program sustainability_metrics
  implicit none

  real :: objects
  real :: relationships

  objects = 10.0
  relationships = 11.0

  print *, "Sustainability Knowledge Architecture Metrics"
  print *, "Metadata coverage: ", 9.0 / objects
  print *, "Justice context coverage: ", 6.0 / objects
  print *, "Uncertainty context coverage: ", 8.0 / objects
  print *, "Relationship traceability: ", 10.0 / relationships
  print *, "Underspecified relationship risk: ", 1.0 / relationships
end program sustainability_metrics
